exercises/object-oriented-programming/src/main/java/com/cbfacademy/cars/Cars.java

```diff
@@ -0,0 +1,70 @@
+package com.cbfacademy.cars;
+
+public class Cars {
+    // public String make = "Volvo";
+    // public String model = "V40";
+    // public int year = 2012;
+     String make = "Volvo";
+     String model = "V40";
+     int year = 2012;
+
+    // public Cars(String make, String model, int year){
+    //     this.make = make;
+    //     this.model = model;
+    //     this.year = year;
+    // }
```

Comment:
Currently, every car instance created will be a 2012 Volvo V40. You should enable this constructor to allow different cars to be made.